.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+on:
+    push:
+        branches: [master]
+    pull_request:
+
+name: Spell Check
+ 
+jobs:
+    codespell:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: codespell-project/actions-codespell@master
+              with:
+                  check_filenames: true
+                  only_warn: 1
+                  ignore_words_list: ba,probleme,formend,compiletime,chage,hve,files,ans,inout,testin,files\',creat
```

Comment:
Why is `hve` being ignored?

Reply:
It wants to change (hVe -> have),

for example:
$options = getopt("dv:n:hVe:j:t:", ["diskspace-error"]);
./judge/judgedaemon.main.php